test/specs/lobbyRejectParticipant.js

```diff
@@ -0,0 +1,36 @@
+import openSession from "../helpers/lobbyHelper"
+import { BASE_URL } from "../helpers/constants"
+
+describe('Activate lobby and reject participant', () => {
+    it('should open jitsi-meet app, enable lobby and reject participant', async () => {
+        await openSession({
+            moderator: true,
+            name: 'Participant 1',
+            url: BASE_URL,
+            roomName: 'RoomNameTest'
+        });
+        await openSession({
+            moderator: false,
+            name: 'Participant 2',
+            url: BASE_URL,
+            roomName: 'RoomNameTest'
+        });
+
+        const handles = await browser.getWindowHandles()
+
+        await browser.switchToWindow(handles[0]);
+        const notification = await $('#notifications-container');
+        await expect(notification).toBeDisplayed();
+        const lobbyAdmitBtn = await $('[data-testid="lobby.admit"]');
+        await expect(lobbyAdmitBtn).toBeDisplayed();
+        const lobbyRejectBtn = await $('[data-testid="lobby.reject"]');
+        await expect(lobbyRejectBtn).toBeDisplayed();
+        await lobbyRejectBtn.click();
+        await browser.switchToWindow(handles[1]);
```

Comment:
Is there a cleaner way to handle this than to mentally keep track that handles[X] is participantX

Reply:
No more tabs. Now we use remote sessions.